src/main/webapp/user_profile.jsp

```diff
@@ -220,6 +220,21 @@
             <span class="close-button" id="task-info-close-button" onclick="closeTaskInfoModal()">&times;</span>
             <h1>Task Detail: </h1>
             <div id="task-detail-container"></div>
+            <hr/>
+            <h1>Chat: </h1>
+            <div id="message-container"></div>
+            <form id="chat-box" action="/messages" method="POST" onsubmit="return validateMessage()">
```

Comment:
This is throwing an error `Uncaught ReferenceError: validateMessage is not defined` - I think the function might be missing from the script file

Reply:
Ah I forgot to add it when I commit the new changes. I will include the function in the new commit.